src/Microsoft.ML.Transforms/NormalizerCatalog.cs

```diff
@@ -51,6 +51,106 @@ public static class NormalizationCatalog
             return new NormalizingEstimator(env, mode, InputOutputColumnPair.ConvertToValueTuples(columns));
```

Comment:
This new method looks good... the new method looks a lot better than the old method. :) I wonder if the old method that just took an "enum" to specify the type of normalization is a good idea. Or if it's already too late to remove. :P Maybe @artidoro has opinion?

Reply:
Since we are adding these new extensions to the stable api, I think it would be a good idea to remove the old method if still possible.

Reply:
Good. Perhaps we could prioritize this @codemzs since this will involve a breaking change in the API, and we must do that immediately?

---
In reply to: [270174092](https://github.com/dotnet/machinelearning/pull/3118#discussion_r270174092) [](ancestors = 270174092)

Reply:
@TomFinley Yes, I should be able to do this in the next hour or so.